src/nvim_bridge.rs

```diff
@@ -211,10 +211,83 @@ pub enum OptionSet {
     NotSupported(String),
 }
 
+#[derive(Clone)]
+pub enum CompletionItemKind {
+    Class,
+    Color,
+    Constant,
+    Constructor,
+    Enum,
+    EnumMember,
+    Event,
+    Function,
+    File,
+    Folder,
+    Field,
+    Interface,
+    Keyword,
+    Method,
+    Module,
+    Operator,
+    Property,
+    Reference,
+    Snippet,
+    Struct,
+    Text,
+    TypeParameter,
+    Unit,
+    Unknown,
+    Value,
+    Variable,
+}
+
+impl From<&String> for CompletionItemKind {
```

Comment:
This should be `From<&str>` instead of `From<&String>`.

Reply:
@vhakulinen After that change, I would need to do something like this in nvim_bridge.rs:
```diff
@@ -753,7 +753,7 @@ fn parse_redraw_event(args: Vec<Value>) -> Vec<RedrawEvent> {
                         let item = unwrap_array!(item);
                         let word = unwrap_str!(item[0]).to_owned();
                         let kind = CompletionItemKind::from(
-                            &unwrap_str!(item[1]).to_owned(),
+                            unwrap_str!(item[1]),
                         );
                         let kind_raw = unwrap_str!(item[1]).to_owned();
                         let menu = unwrap_str!(item[2]).to_owned();
```
Will the above work as intended?